### DISCUSSION TITLE
Ansible for fio-plot

### DISCUSSION THREAD

**Discussion by zhucan** [louwrentius/fio-plot#119] [Ideas]
Because we want to test on more nodes，so we want to support run bench-fio on all nodes with ansible.

**Comment by louwrentius** [comment]
Fio has a client-server mechanism, which allows you to run fio on multiple machines.

Bench-fio recently got support for this mechanism.

The only thing that is not covered is:

1. Installing fio and bench fio
2. Starting fio as a daemon (server mode) on the clients
3. Generating a list of clients for bench-fio to run benchmarks on

Or are you looking for a different use case?

**Reply by zhucan** [reply]
if support ansible, we can install fio and bench-fio with ansible, no need to start fio server manually, it's more convenient.

**Reply by louwrentius** [reply]
If you want to run fio on multiple devices in regular stand-alone / non-client-server mode I can see this. Although I'm an avid Ansible user, I have no particular need for this feature, to be frank.

If you want to contribute/share your code, I'm willing to integrate this, like your previous contribution :)

**Comment by zhucan** [comment]
@louwrentius there is a [demo](https://github.com/zhucan/ansible-tools/tree/master/fio-plot), use fio-plot with ansible
